### DISCUSSION TITLE
A way to allow users to install c-lightning plugins and edit `config`

### DISCUSSION THREAD

**Discussion by fiatjaf** [btcpayserver/btcpayserver#2554] [General]
Plugins are first-class citizens in c-lightning and many of the useful functionalities available are provided as plugins. Without plugins c-lightning is dumb and not fun. But currently installing plugins is a task only for very advanced users experienced with Docker.

Is it possible to conjure a magic way to allow plugins to be installed from the GUI or something?

Also very often plugins require changes in the main `config` file, so editing that should also be necessary, but I imagine that's easier once the above is solved.

**Comment by arcbtc** [comment]
Very much agreed! Having the powerful cln behind btcpayserver without being able to install plugins is tragic
